src/sqlparser/src/ast/mod.rs

```diff
@@ -2631,6 +2630,7 @@ impl fmt::Display for CreateFunctionUsing {
 pub enum SetVariableValue {
     Ident(Ident),
     Literal(Value),
+    List(Vec<SetVariableValue>),
```

Comment:
This recursive type can essentially hold a tree. Maybe:
```
pub enum SetVariableValue {
    Single(SetVariableValueSingle),
    List(Vec<SetVariableValueSingle>),
    Default,
}
pub enum SetVariableValueSingle {
    Ident(Ident),
    Literal(Value),
}
```
Also note that `default` cannot be an element in the list.

Reply:
will fix in next PR